src/python/ddapp/roboturdf.py

```diff
@@ -30,6 +30,7 @@
     handCombinations = directorConfig['handCombinations']
     numberOfHands = len(handCombinations)
 
+    headLink = directorConfig['headLink']
```

Comment:
In this case 'head' means the direction the robot's main sensor is facing which all robots should have.
I'd prefer not to fail like this as its difficult to hunt down the failure.

But you're right about having a limited list of blocks in the cfg file.